### ISSUE TITLE
Ftx fetch_my_trades not filling fee response fully

### ISSUE THREAD

**Issue by xmatthias** [ccxt/ccxt#6821]
- OS: linux (irrelevant)
- Programming Language version: python 
- CCXT version: 1.26.53

When using FTX fetch_my_trades, the fee object is not populated with the `'currency'` key.

Other exchanges (kraken, binance) do so - and the data would be available according to the `'info'` field (`'feeCurrency'`). 

```python
import ccxt
ct = ccxt.ftx()
trades = ct.fetch_my_trades('BNB/USD', since=since)
trades

```

``` output
[{'info': {'baseCurrency': 'BNB',
   'fee': 0.0032162257505,
   'feeCurrency': 'USD',
   'feeRate': 0.00064505,
   'future': None,
   'id': xxxxx,
   'liquidity': 'taker',
   'market': 'BNB/USD',
   'orderId': xxxxx,
   'price': 12.161,
   'quoteCurrency': 'USD',
   'side': 'buy',
   'size': 0.41,
   'time': '2020-03-24T15:27:44.705233+00:00',
   'tradeId': xxxxx,
   'type': 'order'},
  'timestamp': 1585063664705,
  'datetime': '2020-03-24T15:27:44.705Z',
  'symbol': 'BNB/USD',
  'id': 'xxxxx',
  'order': 'xxxxx',
  'type': None,
  'takerOrMaker': 'taker',
  'side': 'buy',
  'price': 12.161,
  'amount': 0.41,
  'cost': 4.986009999999999,
  'fee': {'cost': 0.0032162257505, 'rate': 0.00064505}}
]
```

**Comment by kroitor** [comment]
They have added the fee currency field not so long ago. It was missing previously. Fixed it in CCXT 1.26.55+. Thx for reporting!

**Comment by xmatthias** [comment]
great, thanks for fixing so quickly
